Packs/Emailrep/pack_metadata.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "EmailRep",
+    "description": "Provide email address reputation and reports from EmailRep.io.",
+    "support": "community",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://emailrep.io",
+    "email": "",
+    "created": "2020-03-10T08:37:18Z",
```

Comment:
support should be XSOAR
take a look at Packs/AutoFocus/pack_metadata.json for instance